rotkehlchen/constants/assets.py

```diff
@@ -32,6 +32,7 @@
 A_PETH = Asset('eip155:1/erc20:0xf53AD2c6851052A81B42133467480961B2321C09')
 A_USDT = Asset('eip155:1/erc20:0xdAC17F958D2ee523a2206206994597C13D831ec7')
 A_USDC = Asset('eip155:1/erc20:0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48')
+A_OPTIMISM_USDC = Asset('eip155:10/erc20:0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85')
```

Comment:
I don't see it used but only in tests. Moving it here it is special and you should do it only for production code, not for tests